gtsfm/averaging/rotation/cycle_consistency.py

```diff
@@ -30,6 +31,25 @@
 MAX_INLIER_MEASUREMENT_ERROR_DEG = 5.0
 
 
+class EdgeErrorAggregationCriterion(str, Enum):
```

Comment:
Will we ever need to use min or mean instead of median?

Reply:
So I use `min` in the unit tests (our previous unit tests). Not sure if we would ever use mean. Probably can be removed.

Reply:
i removed `mean`